pkg/manager/manager.go

```diff
@@ -186,42 +190,6 @@ func GetManager() *Manager {
 	return &mgr
 }
 
-// ComputeDeviceChange computes a given device change the given updates and deletes, according to the path
-// on the configuration for the specified target
-func (m *Manager) ComputeDeviceChange(deviceName devicetype.ID, version devicetype.Version,
```

Comment:
I just moved this to `setconfig.go` in the same package - it's only ever used from there so did not need to be in this file